LoopMusicViewController.m

```diff
@@ -115,6 +115,56 @@ - (void)viewDidLoad
         [self updatePlaylistName];
     }
     
+// Load local test files from the testSongs/ directory if testing on the simulator
+#if TARGET_OS_SIMULATOR
+    NSString *localDir = @"testSongs/";
+    if(![self isSongListEmpty])
```

Comment:
Please follow the coding style of this repository. Space between "if" and parentheses.